handler/proxy.go

```diff
@@ -99,11 +99,16 @@ func (c *CORSOptions) NeedsVary() bool {
 }
 
 func (c *CORSOptions) AllowsOrigin(origin string) bool {
+	if c == nil {
```

Comment:
Currently never happens in the code.

Reply:
yep, I had this discussion already ... on Test debugging I am getting a panic every time if this statement is missing even if this option obj is provided.

Reply:
OK, it doesn't hurt if it stays in...